app/views/matches/_current_step_info.haml

```diff
@@ -15,13 +15,34 @@
             = @match.current_decision.try(:actor_type) || 'N/A'
       = render 'match_list_base/current_step', match: @match
   - else
-    %p This match is not active
+    -  description = 'This match is not active'
+    - if can_reopen_matches? && ! @match.can_be_reopened?
+      - description += ', and cannot be re-opened because'
```

Comment:
Can we move the calculation of what `description` is into the model?  It becomes much more complicated than it needs to be in the view.  Also, if you treat description as an array and just join it with `, and` at the you won't need to keep track of `emit_and`